packages/components/src/advanced-filters/stories/index.js

```diff
@@ -149,7 +149,7 @@ const advancedFilters = {
 	},
 };
 
-export default () => (
+export const Basic = () => (
```

Comment:
Migration is a lot of copy and paste work, I've kept the story migration very simple for now and it usually involves exporting a `Basic` example that renders the original devdocs example. This would definitely need more improvement at some stage.